pv_site_api/main.py

```diff
@@ -111,21 +119,7 @@ def get_sites(
         print(site.client)
         print(site.client.client_name)
 
-        pv_sites.append(
-            PVSiteMetadata(
-                site_uuid=str(site.site_uuid),
-                client_name=site.client.client_name,
-                client_site_id=site.client_site_id,
-                client_site_name=site.client_site_name,
-                region=site.region,
-                dno=site.dno,
-                gsp=site.gsp,
-                latitude=site.latitude,
-                longitude=site.longitude,
-                installed_capacity_kw=site.capacity_kw,
-                created_utc=site.created_utc,
-            )
-        )
+        pv_sites.append(site_to_pydantic(site))
```

Comment:
how come you have this `site_to_pydantic` function? I think `PVSiteMetadata` is already pydantic object